### PR TITLE
Add bike rental smoke test

### PR DESCRIPTION
**Pull request by leonardehrenfried**
The recent bike rental breakage at TriMet reminded my that I should have added one more smoke test, namely for planning an actual bike rental route.

The current smoke test just checks that the API returns rental stations which is not enough.

### CODE REVIEW

File: src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java [hunk 1]
```diff
@@ -8,8 +8,6 @@
 
 /**
  * This smoke test expects an OTP installation running at localhost:8080
- * <p>
- * It uses the REST API to check that a route from Philadelphia's airport to its North can be found.
```
Comment (miles-grant-ibigroup) [inline src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java:12]: Why was this removed?

**Review by miles-grant-ibigroup — APPROVED** [review]
Clever test! I like using directions to see if the stations loaded. Just one quick question about a removed comment

File: src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java (hunk 1 above)
Comment (leonardehrenfried) [reply src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java:12]: It was outdated and didn't reflect anymore the content of the test. I didn't want to repeat the list of tests below either. Did you find it useful?

**Review by br648 — APPROVED** [review]
LGTM.

File: src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java (hunk 1 above)
Comment (miles-grant-ibigroup) [reply src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java:12]: Makes sense. Can be removed!
